unlock-app/src/components/content/AccountContent.tsx

```diff
@@ -141,7 +150,26 @@ export const mapDispatchToProps = (dispatch: any) => ({
   dismissPurchaseModal: () => dispatch(dismissPurchaseModal()),
 })
 
-export default withConfig(connect(mapStateToProps)(AccountContent))
+export const getStripeHelper = (
```

Comment:
Would it be worth moving to its own file?

Reply:
At the moment this is a small enough thing that I think it's ok to leave it here. If we ever start using Stripe across more than one page, then I think it's definitely worth pulling out.